docs/user/admin.rst

```diff
@@ -0,0 +1,204 @@
+Admin
+===============
+
+Installing Kolibri
+------------------
+
+Windows
+~~~~~~~~
+
+To install Kolibri on Windows follow these steps:
+
+#. Download the Kolibri Windows installer. **(TODO - link)**
+#. Double-click the downloaded ``.exe`` file.
+#. Follow the instructions in the installation wizard window.
+
+
+Creating a Device Owner Account
+-------------------------------
+
+In order to manage Kolibri content channels, data and users, you must first create your own **Device Owner** account and **Facility**. The **Device Owner** account registration page appears when you open the browser at http://127.0.0.1:8000/ after the installation of Kolibri.
+
+.. image:: img/wizard.png
+  :alt: create device owner form
+
+#. Enter the required information for your **Device Owner** account (username, password, name of the facility). Fields marked with an asterisk (*) are required.
+#. Select **Create and get started**.
+
+
+Adding a New Device Owner
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Device Owner** is a “super-user” for your Kolibri installation. In case you need to create another user with this kind of permissions, run the following command in the ``cmd.exe`` (Windows command prompt) window:
+
+.. code-block:: bash
+
+  kolibri manage createsuperuser
+
+You will be prompted to input the **Username** and **Password** and the new **Device Owner** user account will be created.
+
+
+Manage Users
+------------
+
+.. note::
+  To manage Kolibri users you must be logged-in as **Device Admin**. As **Device Admin** you can manage all users: *Learners* and other *Admins*.
```

Comment:
Wow, thanks for the detailed explanation, just the type of SME feedback tech writers dream about... :heart_eyes_cat: